aws/session/session.go

```diff
@@ -210,6 +211,12 @@ type Options struct {
 	// the config enables assume role wit MFA via the mfa_serial field.
 	AssumeRoleTokenProvider func() (string, error)
 
+	// When the SDK's shared config is configured to assume a role with MFA
+	// this option may be provided to set the expiry duration of the STS
+	// credentials. Defaults to 15 minutes if not set as documented in the
+	// stscreds.AssumeRoleProvider.
+	AssumeRoleTokenDuration time.Duration
```

Comment:
Since the `AssumeRoleTokenDuration` can be used with any Assume Role. I think this should be renamed to `AssumeRoleDuration`. This will help prevent confusion with assume Role with or without MFA. The `Token` in AssumeRoleTokenProvider referred to the MFA token.

In addition, the documentation for this member should clarify that it is for all assume roles performed by the Session not just MFA. The SDK is investigating how to support multiple assume role chaining, and this option would need to apply to all of them.